### PR TITLE
Update Ukrainian localization

### PR DESCRIPTION
**Pull request by IllusiveMan196**

### CODE REVIEW

File: resources/l10n/uk_UA/LC_MESSAGES/loot.po [hunk 1]
```diff
@@ -14,336 +14,336 @@ msgstr ""
...
+#: src/gui/qt/main_window.cpp:667
+msgid "&View Documentation"
+msgstr "&Відкрити документацію"
 
-#: src/gui/qt/main_window.cpp:614
-msgid "Join Discord Server"
-msgstr "Приєднатися до серверу Discord"
+#: src/gui/qt/main_window.cpp:668
+msgid "&Join Discord Server"
+msgstr "Приєднатися до серверу &Discord"
```
Comment (sibir-ine) [inline resources/l10n/uk_UA/LC_MESSAGES/loot.po:288]: This is a Latin `D` as well, right? Is this also on a Cyrillic keyboard?

**Review by sibir-ine — CHANGES_REQUESTED** [review]

**Comment by IllusiveMan196** [comment]
As odd as it may be, but yes, both Latin and Cyrillic letters are on the
keyboard (unless it's some odd case where there is only one set, arguably
some keyboards have just baf execution of one or the other letter sets, so
despite having both technically... you have just one with good visibility).
Most shortcuts are also in Latin (courtesy of technology being made with
English-centric UI), so... technically doesn't really matter what key is
assigned to them.

As for mnemonics and & signs - feel free to redesignate them as I've
essentially played a game of "hope this letter isn't used anywhere else"
with them. Apparently not always successfully.

**Comment by sibir-ine** [comment]
> As for mnemonics and & signs - feel free to redesignate them as I've essentially played a game of "hope this letter isn't used anywhere else" with them. Apparently not always successfully.

Re conflicting mnemonics, see https://github.com/loot/loot/pull/1778#issuecomment-1214810622 about how to tell if multiple conflict. Ideally, if the first character in a string has already been taken, the next most sensible character that hasn't also been taken would be used as opposed to any random character in the string, & that's not something we can really determine without being able to read Ukrainian. If you still have trouble assigning mnemonics without them conflicting we can try to guess at optimal assignments, but we'd appreciate it if you could give it another try first after I've hopefully clarified things. Also, you're always welcome to ask for further clarification if needed.

**Comment by Ortham** [comment]
I've merged #1780 so if you merge/rebase you should now see comments in Poedit that tell you which strings appear in which menus, which should help avoid conflicts.

File: resources/l10n/uk_UA/LC_MESSAGES/loot.po [hunk 2]
```diff
@@ -14,336 +14,372 @@ msgstr ""
...
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:667
+msgid "Copy &Load Order"
+msgstr "Скопіювати порядок &завантаження"
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:669
+msgid "&Copy Content"
+msgstr "&Скопіювати зміст"
```
Comment (sibir-ine) [inline resources/l10n/uk_UA/LC_MESSAGES/loot.po:252]: This conflicts with `&Cортувати плагіни`. Anything other than `Г`, `C`, `П`, `А`, `К`, `Р`, `З`, `О`, `Д`, `В`, or `І` would be fine.

File: resources/l10n/uk_UA/LC_MESSAGES/loot.po [hunk 3]
```diff
@@ -14,336 +14,372 @@ msgstr ""
...
+
+#. translators: This string is an action in the Plugin menu.
+#: src/gui/qt/main_window.cpp:685
+msgid "Copy &Card Content"
+msgstr "Скопіювати зм&іст картки"
+
+#. translators: This string is an action in the Plugin menu.
+#: src/gui/qt/main_window.cpp:687
+msgid "Copy &Metadata"
+msgstr "&Копіювати метадані"
```
Comment (sibir-ine) [inline resources/l10n/uk_UA/LC_MESSAGES/loot.po:293]: This now conflicts with `&Копіювати назву плагіну`. Anything other than `П`, `К`, `І`, `У`, or `Т` would be fine.

File: resources/l10n/uk_UA/LC_MESSAGES/loot.po [hunk 4]
```diff
@@ -14,336 +14,372 @@ msgstr ""
...
+
+#. translators: This string is an action in the Help menu.
+#: src/gui/qt/main_window.cpp:697
+msgid "&View Documentation"
+msgstr "&Відкрити документацію"
+
+#. translators: This string is an action in the Help menu.
+#: src/gui/qt/main_window.cpp:699
+msgid "&Join Discord Server"
+msgstr "Приєднатися до сер&веру Discord"
```
Comment (sibir-ine) [inline resources/l10n/uk_UA/LC_MESSAGES/loot.po:319]: This now conflicts with `&Відкрити документацію`. Anything other than `Д`, `В`, `Р`, or any Latin characters would be fine.

File: resources/l10n/uk_UA/LC_MESSAGES/loot.po [hunk 5]
```diff
@@ -14,336 +14,336 @@ msgstr ""
...
+#: src/gui/qt/main_window.cpp:650
+msgid "&Edit Groups..."
+msgstr "&Редагувати групи..."
 
-#: src/gui/qt/main_window.cpp:597
-msgid "Search Cards..."
-msgstr "Знайти картки..."
+#: src/gui/qt/main_window.cpp:651
+msgid "Searc&h Cards..."
+msgstr "Знайти &картки..."
```
[the remark this answers is not in the document]
Comment (sibir-ine) [reply resources/l10n/uk_UA/LC_MESSAGES/loot.po:242]: This still conflicts with `С&касувати відсортований порядок завантаження`. Anything other than `Г`, `І`, `П`, `У`, `К`, `Р`, `В`, `С`, `О`, `Е`, or `Ч` would be fine.

File: resources/l10n/uk_UA/LC_MESSAGES/loot.po [hunk 6]
```diff
@@ -14,336 +14,372 @@ msgstr ""
...
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:673
+msgid "Redate &Plugins..."
+msgstr "П&ередатувати плагіни..."
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:675
+msgid "&Fix Ambiguous Load Order"
+msgstr "&Виправити неоднозначний порядок завантаження"
```
Comment (sibir-ine) [inline resources/l10n/uk_UA/LC_MESSAGES/loot.po:267]: This now conflicts with `Скопію&вати порядок завантаження`. Anything other than `Г`, `І`, `П`, `У`, `К`, `Р`, `В`, `С`, `О`, `Е`, or `Ч` would be fine.

**Comment by IllusiveMan196** [comment]
Improved, hopefully no collisions between symbols this time.

**Review by sibir-ine — APPROVED** [review]
Thanks! The mnemonics look good now.

**Comment by IllusiveMan196** [comment]
Note to myself - don't merge anything while PR is ongoing.

**Comment by sibir-ine** [comment]
It's because you submitted a PR from your master branch. If you made a new branch specifically for your PR, you could sync it separately & only when needed. It'd also be easier to keep your master branch clean & even with the upstream branch.

**Comment by Ortham** [comment]
Thanks 👍

**Pull request merged**